Resources/Prototypes/DeltaV/Reagents/Consumable/Drink/drinks.yml

```diff
@@ -205,3 +205,19 @@
     Poison:
       effects:
       - !type:Jitter
+
+- type: reagent
+  id: DoubleIceCream
+  name: reagent-name-double-ice-cream
+  parent: BaseSoda
+  desc: reagent-desc-double-ice-cream
+  physicalDesc: reagent-physical-desc-fizzy-and-creamy
+  flavor: doubleicecream
+  color: "#a06f42"
+  metamorphicSprite:
+    sprite: Objects/Consumable/Drinks/doubleicecreamglass.rsi
```

Comment:
This should path to DeltaV/Objects/Consumable/Drinks/doubleicecreamglass.rsi

Reply:
alright, should be done now